stdlib/public/core/SequenceAlgorithms.swift.gyb

```diff
@@ -353,12 +353,14 @@ ${equivalenceExplanation}
   public func elementsEqual<OtherSequence>(
     _ other: OtherSequence${"," if preds else ""}
 %   if preds:
-    by areEquivalent: (${GElement}, ${GElement}) throws -> Bool
+    by areEquivalent: (${GElement}, OtherSequence.${GElement}) throws -> Bool
 %   end
   ) ${rethrows_}-> Bool
     where
-    OtherSequence: Sequence,
+    OtherSequence: Sequence${" {" if preds else ","}
+%   if not preds:
     OtherSequence.${GElement} == ${GElement} {
```

Comment:
@nkcsgexi It is concerning that the change in the generic requirements was not caught by swift-api-digester.